package.json

```diff
@@ -97,7 +104,10 @@
       "fsevents": false,
       "core-js-pure": false,
       "esbuild": false,
-      "@swc/core": false
+      "@swc/core": false,
+      "tsup>esbuild": true,
+      "@storybook/react-webpack5>@storybook/preset-react-webpack>@pmmmwh/react-refresh-webpack-plugin>core-js-pure": false,
+      "tsup>@swc/core": false
     }
   }
 }
```

Comment:
Here are the prompts I used to achieve parity between the [module-template package.json](https://github.com/MetaMask/metamask-module-template/blob/354995b0a4040cbb74f07b42a0fbc279fb5a1ea0/package.json) and the design-tokens package.json

<img width="866" alt="Screenshot 2024-01-30 at 4 17 38 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/6e786709-e181-408a-a8cb-0b10158085c4">
<img width="837" alt="Screenshot 2024-01-30 at 4 17 48 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/48285987-183e-42cc-915c-b1dcfb08f456">
<img width="876" alt="Screenshot 2024-01-30 at 4 17 57 PM" src="https://github.com/MetaMask/design-tokens/assets/8112138/93fb4b96-635a-4c6a-9737-1adf20729450">

I then compared and made some small tweaks like adding some missing storybook dependecies that were dropped and removing typedoc as we use storybook for our documentation